_opensearch/snapshot-restore.md

```diff
@@ -156,8 +156,23 @@ Setting | Description
    s3.client.default.proxy.port: 8080 # port for your proxy server
    s3.client.default.read_timeout: 50s # the S3 connection timeout
    s3.client.default.use_throttle_retries: true # whether the client should wait a progressively longer amount of time (exponential backoff) between each successive retry
+   s3.client.default.region: us-east-2 # AWS region to use
    ```
 
+1. (Optional) If you don't want to use AWS access and secret keys, you could configure S3 plugin to use AWS IAM roles for service accounts:
+
+   ```bash
+   sudo ./bin/opensearch-keystore add s3.client.default.role_arn
+   sudo ./bin/opensearch-keystore add s3.client.default.role_session_name
+   ```
+
+   If you don't want to configure AWS access and secret keys, modify the following `opensearch.yml` setting. Make sure the file is accessible by the `repository-s3` plugin:
+   ```yml
+   s3.client.default.identity_token_file: /usr/share/opensearch/plugins/repository-s3/token
+   ```
+
+   AWS IAM roles require at least one of the above settings. Other settings will be taken from environment variables (if available): `AWS_ROLE_ARN`, `AWS_WEB_IDENTITY_TOKEN_FILE` and `AWS_ROLE_SESSION_NAME`.
```

Comment:
Change "AWS IAM" to "IAM". Insert a comma after "'AWS_WEB_IDENTITY_TOKEN_FILE'".